packages/tpetra/core/test/CrsMatrix/CrsMatrix_UnitTests3.cpp

```diff
@@ -422,9 +422,6 @@ inline void tupleToArray(Array<T> &arr, const tuple &tup)
 
     // call fillComplete()
     out << "Call fillComplete on the matrix" << endl;
-#ifdef TPETRA_ENABLE_DEPRECATED_CODE
-    TEST_EQUALITY_CONST( A.getProfileType() == Tpetra::DynamicProfile, true );
```

Comment:
Why just delete this one, but test with `TPETRA_DEFAULT_PROFILE_TYPE` above?

Reply:
This test seemed minimally important, and would go away with the deprecated code anyway.